packages/apps/composer-app/src/plugins/SpacePlugin/components/SpaceMain.tsx

```diff
@@ -4,15 +4,18 @@
 
 import React, { FC } from 'react';
 
+import { Document } from '@braneframe/types';
 import { useTextModel } from '@dxos/aurora-composer';
-import { SpaceProxy } from '@dxos/client';
+import { isTypedObject, SpaceProxy } from '@dxos/client';
 import { observer } from '@dxos/observable-object/react';
 import { useIdentity } from '@dxos/react-client';
 import { Surface, useGraphContext, useTreeView } from '@dxos/react-surface';
 
-import { isDocument } from '../../GithubMarkdownPlugin';
 import { SpacePlugin } from '../SpacePlugin';
 
+export const isDocument = (datum: unknown): datum is Document =>
```

Comment:
@dmaretskyi how would you feel about including these helpers with the echo-schema generated code?